submissions/report.Rmd

```diff
@@ -0,0 +1,104 @@
+---
+title: "US forecasts: weekly review"
+output:
+  html_document:
+    toc: true
+    toc_float:
+      collapsed: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(here)
+library(dplyr)
+source(here::here("submissions", "finalise.R"))
+source(here("utils", "get-us-data.R"))
+source(here("utils", "load_observations.R"))
+source(here("evaluation", "utils", "plot_data.R"))
+
+target_date <- as.Date(readRDS(here::here("data", "target_date.rds"))) 
+
+# Change these options to alter the final submission
+ensemble = "QRA"
+exclude_locations = NULL
+```
+
+# `r target_date`
+
+## Evaluation {.tabset}
+
+### Data {.tabset .tabset-pills}
+
+```{r setup-data, echo=FALSE, fig.align = 'center', out.width="100%", out.height="100%", warning=FALSE, message=FALSE}
+weekly <- load_observations(target_date) 
+
+daily <- get_us_deaths("daily") %>%
+  filter(date >= min(weekly$date)) %>%
+  select(state, date, value = deaths)
+
+daily_us <- daily %>%
```

Comment:
how many times do we work this out? We should probably have it in the data call and just do it once.